pkg/apis/marketplace/v1alpha1/catalogsourceconfig_types.go

```diff
@@ -13,15 +13,8 @@ const (
 	CSCFinalizer = "finalizer.catalogsourceconfigs.marketplace.redhat.com"
 )
 
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
I had to move this for `client-gen` to work. I ran `operator-sdk generate k8s` after this and it did not throw any error or produce any changes to the typeset.